src/language-markdown/pragma.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const pragmas = ["format", "prettier"];
+const startWithPragmaRegex = new RegExp(
+  `^<!--\\s*@(${pragmas.join("|")})\\s*-->`
+);
+
+module.exports = {
+  startWithPragmaRegex,
+  hasPragma: text => startWithPragmaRegex.test(text.trimLeft()),
```

Comment:
Do we need to worry about this case?

Reply:
Or:
```md
<!--
  MIT LICENSE

  bla bla bla
-->

<!-- @format -->
```

Reply:
We should worry about the first one, but I'm not sure about the second one, maybe just doing something like JS?
```md
<!--
  MIT LICENSE 
  
  bla bla bla

  @format
-->
```

Reply:
That looks good to me :+1: